### DISCUSSION TITLE
Are there any settings to scrap posts or comments of a certain language? Something like lang='fil'

### DISCUSSION THREAD

**Discussion by 22033662-CWM** [kevinzg/facebook-scraper#840] [Q&A]
Hi, I am trying to scrape comments and posts in Filipino for class work. Any settings for that?
Thanks in advance!

**Comment by neon-ninja** [comment]
No, it should just work

**Reply by 22033662-CWM** [reply]
Thank you for replying and for the very usable fb scraper!
